### ISSUE TITLE
[Feature]: Learn about PyTorch & Neural Networks

### ISSUE THREAD

**Issue by mvallenet** [PoCInnovation/MindGame#2]
### Description

Complete the POC & 42AI technical exercise to gain skills in neural networks.

### Expected behaviour

The jupyter notebook must be completed and documented on the methods used.
The students must be able to explain the basic functioning of a neural network.

- [x] The exercise is successful
- [x] The Notebook is documented
- [x] Visualization plots on loss and accuracy have been developed

### Environment

/

### Proposed solution

/

**Comment by Baragouin** [comment]
On va s'occuper de faire un beau notebook avec @agherasie avant de passer sur la #3. Ce serait possible d'avoir la correction que tu nous a montré toute à l'heure @Mikatech ?

**Comment by mvallenet** [comment]
> On va s'occuper de faire un beau notebook avec @agherasie avant de passer sur la #3. Ce serait possible d'avoir la correction que tu nous a montré toute à l'heure @Mikatech ?

Done sur discord 🚀

**Comment by mvallenet** [comment]
Task is done [here](https://github.com/Baragouin/MNIST-Fashion) 🚀
